### PR TITLE
refactor(Containers): move Containers reducer

### PR DESCRIPTION
**Pull request by nLight**
Saving https://github.com/dcos/dcos-ui/pull/2061

What I did:
- Squash / prettier / #SRC / basic conflicts
- Kept the original branch `ph/refactor/DCOS-14525-refactor-containers`

@Poltergeist: 
- I kept both Containers(-test).js we need to merge them. Because it looks like you've done some bug fixing along the way

rescue-rangers-logo

### CODE REVIEW

**Comment by weblancaster** [comment]
@nLight the changes looks good and seems like it's working crrectly but _failing unit tests_.

**Comment by nLight** [comment]
@weblancaster sorry I forgot to explicitly mention that this is a WIP. We still need to reintegrate changes made in `serviceForm/Containers.js` to `serviceForm/JSONReducers/Containers.js` We will have a look together with @Poltergeist

**Comment by orlandohohmeier** [comment]
@nLight @Poltergeist what do we need to do to land this refactoring?

**Comment by nLight** [comment]
@orlandohohmeier merge Containers.js and delete an obsolete one. And resolve conflicts :)

**Comment by orlandohohmeier** [comment]
@Poltergeist could you please take care of the respective tasks?

**Comment by nLight** [comment]
I'd suggest we team up with @Poltergeist and fix this PR tomorrow

**Comment by nLight** [comment]
flicts

**Comment by nLight** [comment]
rebasing...

**Comment by nLight** [comment]
It's good to review with that one

**Comment by orlandohohmeier** [comment]
@Poltergeist please rebase to include the latest changes and fixes (e.g. volume size).

**Comment by Poltergeist** [comment]
@orlandohohmeier done.

**Comment by nLight** [comment]
run integration tests

File: plugins/services/src/js/reducers/JSONSingleContainerParser.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import {
   JSONParser as constraints
 } from "./serviceForm/JSONReducers/Constraints";
 import { JSONParser as container } from "./serviceForm/Container";
-import { JSONParser as containers } from "./serviceForm/Containers";
```
Comment (weblancaster) [inline plugins/services/src/js/reducers/JSONSingleContainerParser.js]: WHAT?!

Comment (Poltergeist) [inline plugins/services/src/js/reducers/JSONSingleContainerParser.js]: Sorry, I missed to explain this is the singleContainerParser. `containers` is a attribute only available in multipleContainers meaning this parser was running on every singleContainerParser without any purpose. Meaning we removed at least one function call here. Which also improves the performance.

Comment (weblancaster) [inline plugins/services/src/js/reducers/JSONSingleContainerParser.js]: Sorry my `WHAT` was to express why we were importing same file two times but I missed the `s` at the end.

**Pull request merged**